src/System.Net.WebSockets.WebSocketProtocol/src/System/Net/WebSockets/WebSocketProtocol.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Threading;
+
+namespace System.Net.WebSockets
+{
+    public static class WebSocketProtocol
+    {
+        public static WebSocket CreateFromConnectedStream(
+            Stream stream,
+            bool isServer,
+            string subProtocol,
+            TimeSpan keepAliveInterval,
+            ArraySegment<byte> buffer)
```

Comment:
This should have the same signature as the method on WebSocket:
```C#
Stream stream, bool isServer, string subProtocol, TimeSpan keepAliveInterval, Memory<byte> buffer = default
```
That means the last parameter should be a `Memory<byte>` instead of an `ArraySegment<byte>`, and should have a default value.